storage/innobase/log/log0log.cc

```diff
@@ -1143,7 +1143,7 @@ log_read_encryption() {
 		/* Make sure the keyring is loaded. */
 		if (!Encryption::check_keyring()) {
 			ut_free(log_block_buf_ptr);
-			ib::error() << "Redo log was encrypted,"
+			ib::fatal() << "Redo log was encrypted,"
```

Comment:
I like ib::fatal() here and elsewhere in the patch, just am wondering whether the previous ib::error()/return(false); resulted in eventual server startup error too?

Reply:
It did, as in this case, this method returns an error and innodb initialization aborts.